docs/conf/prague/2019/news/announcing-schedule.rst

```diff
@@ -0,0 +1,41 @@
+:template: {{year}}/generic.html
+
+.. post:: August 1, 2019
+   :tags: 2019, prague-2019, schedule
+
+Announcing Full Schedule & Tickets Almost Sold Out
+==================================================
+
+Although it hasn't been that long since our last update, we have some big news to share with you all!
+
+Tickets almost sold out
+-----------------------
+
+Since our last blog post, we had a surge of registrations, and we are now **more than 80% sold out**. This means that we only have a few dozen tickets left, and we expect to reach our capacity within the next week or so.
+
+If you're still considering whether to attend, now is definitely the time to `buy your ticket <https://www.writethedocs.org/conf/{{shortcode}}/{{year}}/tickets/>`_.
+
+As a reminder, we cannot be held responsible for any loss of non-refundable travel expenses, and we cannot risk overselling the venue capacity. Please make sure you have a conference ticket before you book your travel!
+
+After we reach capacity, we will operate a waiting list on the ticket website in case of any cancellations, but historically we haven't had many cancellations to our events, so we cannot guarantee that anyone can be taken off the waiting list.
+
+Full presentation schedule
+--------------------------
+
+The conference website is now updated with the `talk schedule <https://www.writethedocs.org/conf/prague/2019/schedule/>`_ for the main stage presentations, so you can start planning your time (and getting excited!) for the conference.
+
+Parallel to the conference talks, we will run the unconference **all day on Monday and in the afternoon on Tuesday**. On Tuesday morning the unconference room will be converted to the **Job Fair**, where you can meet companies who care about content and find out about documentarian-friendly jobs all around Europe and the world.
+
+We'll also have a round of lightning talks after lunch on each conference day.
+You'll be able to sign up for unconference sessions and lightning talks during the conference.
+
+Conference shirts are still on sale
+-----------------------------------
+
+Another friendly reminder that our `T-Shirt Pop-Up Shop <https://teespring.com/wtd-prague-2019>`__ is still open! The campaign will run until September 30th in case anyone wants to order their shirts after the conference, but we recommend to order at least 2-3 weeks before the conference if you want to sport your new threads.
+
+And don't worry, you'll still have a Write the Docs Prague 2019 souvenir from the conference even if you decide not to order a shirt.
+
+Thanks so much for all your support and enthusiasm so far – we're so excited to see you all next month!
+
```

Comment:
We should thank all of our sponsors here. We have a few new ones as well, but I think we should just thank them all. 

I need to add them to the website, let me do that quickly :)

Reply:
Sponsors added, merge pending your PR